lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -212,7 +212,7 @@ defmodule Bamboo.SMTPAdapter do
   end
 
   defp format_email({nil, email}), do: email
-  defp format_email({name, email}), do: "<#{email}> #{name}"
+  defp format_email({name, email}), do: "#{name}<#{email}>"
   defp format_email(emails) when is_list(emails) do
```

Comment:
Any chance this is causing failures with some SMTP servers? The ordering is now correct, but you've removed the space between the two.